modules/video/docs/api-reference/video-builder.md

```diff
@@ -0,0 +1,43 @@
+# VideoBuilder
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v3.2-blue.svg?style=flat-square" alt="From-v3.2" />
+</p>
+
+> The `VideoBuilder` is experimental.
+
+A basic WebM Video encoder. Only works in the browser.
+
+| Encoder        | Characteristic                                          |
+| -------------- | ------------------------------------------------------- |
+| File Extension | `.webm`                                                 |
+| File Type      | Binary                                                  |
+| File Format    | WebM                                                    |
+| Data Format    | `Video` (browsers) (Not currently supported on node.js) |
+| Supported APIs | `addFrame`, `finalize`                                  |
+
+## Usage
+
+```js
+import {VideoBuilder} from '@loaders.gl/video';
+
+const videoBuilder = new VideoBuilder({ source: canvas, frameRate: 24 });
+
+for (const frame of frames) {
+  ctx.drawImage(frame);
+  videoBuilder.addFrame(ctx);
+}
+
+const url = await videoBuilder.finalize();
+video.src = url;
+```
+
+## Options
+
+| Option | Type | Default | Description |
+| ------ | ---- | ------- | ----------- |
+| `width` | number | 512 | The height of the output video
+| `height` | number | 512 | The width of the output video
+| `framerate` | number | 24 | How many frames per second the final container should hold
+| `bitrate` | number | 200 | The bitrate (in kbps) to record the
+| `realtime` | boolean | false | Whether to record in realtime. Used for streaming videos, for instance.
```

Comment:
Add an attribution at the bottom perhaps if we include the wasm see below.

Reply:
Good call. Done.